docs/events.md

```diff
@@ -0,0 +1,88 @@
+| WAITING |  waiting| Fires when the video stops because it needs to buffer the next frame
+
+## Custom events list
+|  Event Name| Event String | Description |
+|--|--|--|
+| ENTER_FULLSCREEN | enterfullscreen| Fires when the player enters fullscreen |
+| EXIT_FULLSCREEN | exitfullscreen | Fires when the player exits fullscreen |
+| REQUESTED_ENTER_FULLSCREEN | requestedenterfullscreen | Fires when the player received a request to enter fullscreen |
+| REQUESTED_EXIT_FULLSCREEN |  requestedexitfullscreen|Fires when the player received a request to exit fullscreen  |
+| AUTOPLAY_FAILED | autoplayfailed | Fires when browser fails to autoplay with sound |
+| FALLBACK_TO_MUTED_AUTOPLAY | fallbacktomutedautoplay | Fires when browser fails to autoplay with sound but start muted autoplay instead |
+| CHANGE_SOURCE_STARTED | changesourcestarted | Fires when change source flow started |
+| CHANGE_SOURCE_ENDED |  changesourceended| Fires when change source flow ended |
+| MUTE_CHANGE | mutechange | Fires when the volume has been muted/unmute |
+| VIDEO_TRACK_CHANGED | videotrackchanged | Fires when the active video track has been changed |
+| AUDIO_TRACK_CHANGED | audiotrackchanged | Fires when the active audio track has been changed |
+| TEXT_TRACK_CHANGED | texttrackchanged | Fires when the active text track has been changed |
+| TEXT_CUE_CHANGED | textcuechanged |Fires when the active text track cue has changed  |
+| TRACKS_CHANGED | trackschanged | Fires when the player tracks have been changed |
+| ABR_MODE_CHANGED | abrmodechanged | Fires when the abr mode change from 'auto' to 'manual' or vice versa|
+| ABR_MODE_CHANGED | abrmodechanged | Fires when the abr mode change from 'auto' to 'manual' or vice versa |
+| PLAYER_STATE_CHANGED | playerstatechanged | Fires when the player state has been changed |
+|FIRST_PLAY  |  firstplay| Fires on the first play |
+| SOURCE_SELECTED | sourceselected | Fires when the player has selected the source to play |
+| TEXT_STYLE_CHANGED | textstylechanged | Fires when the text track style has changed |
+| AD_LOADED | adloaded | Fired when ad data is available. |
+| AD_STARTED | adstarted |  Fired when the ad starts playing|
+|  AD_RESUMED| adresumed | Fired when the ad is resumed |
+|  AD_PAUSED| adpaused | Fired when the ad is paused |
+| AD_CLICKED | adclicked |  Fired when the ad is clicked|
+| AD_SKIPPED | adskipped | Fired when the ad is skipped by the user |
+| AD_COMPLETED | adcompleted | Fired when the ad completes playing |
+| AD_ERROR | aderror | Fired when an error occurred while the ad was loading or playing |
+| ALL_ADS_COMPLETED | alladscompleted | Fired when the ads manager is done playing all the ads |
+| AD_BREAK_START | adbreakstart | Fired when content should be paused. This usually happens right before an ad is about to cover the content |
+| AD_BREAK_END | adbreakend | Fired when content should be resumed. This usually happens when an ad finishes or collapses |
```

Comment:
Fired when content should be resumed. This usually occurs when an ad finishes or collapses